lib/beaker/tasks/rake_task.rb

```diff
@@ -79,13 +81,21 @@ def check_for_beaker_type_config
       end
 
       #
-      # Check for existence of ENV variables for test if !@tests is undef
+      # Check for existence of ENV variables for
+      # tests, hosts, preserver_hosts, fail_mode, provision and log_level
+      # if they are not previously set by the calling rake task.
+      #
       #
       def check_env_variables
         if File.exists?(File.join(DEFAULT_ACCEPTANCE_ROOT, 'tests'))
           @tests = File.join(DEFAULT_ACCEPTANCE_ROOT, 'tests')
         end
         @tests = ENV['TESTS'] || ENV['TEST'] if !@tests
+        @hosts = ENV['BEAKER_setfile'] || ENV['BEAKER_set'] if !@hosts
```

Comment:
We will be merging beaker and beaker-rspec in the near future.  For now, instead of increasing env var clutter, let's just support a single env var that points to a file of a configuration hash - then send it to beaker with --options-file.